libbpf-tools/runqslower.c

```diff
@@ -64,6 +68,13 @@ static error_t parse_arg(int key, char *arg, struct argp_state *state)
 	case 'v':
 		env.verbose = true;
 		break;
+	case 'c':
+		env.comm = strdup(arg);
+		len = strlen(arg);
+		if (len >= TASK_COMM_LEN)
```

Comment:
If the len variable is only used once, is it really necessary to declare it? I'm curious about your thoughts.